service_notification/src/main/kotlin/org/thechance/service_notification/endpoints/userRoutes.kt

```diff
@@ -23,12 +24,18 @@ fun Route.userRoutes() {
             val result = useCase.createUser(user.toEntity())
             call.respond(HttpStatusCode.Created, "User was created: $result")
         }
+        post("/{userId}/token") {
+            val userId = call.parameters["userId"] ?: throw BadRequestException("User id is required")
```

Comment:
Can you improve the name of exception,  Hmmm  to MissingParameterException for example,i think is better than BadRequestException
oh and i see is better that using error code that be Int not string like this "User id is required"

Reply:
Still gonna handle the validatin and errors.